R/load_files.R

```diff
@@ -1078,17 +1078,17 @@ read_in_results_Tecan <- function(results_file, results_sheets, headers) {
     # rows and columns in data matrix with row and col names
     n_row <- nrow(dfm)
     n_col <- ncol(dfm)
-    readout <- as.data.frame(dfm[2:n_row, 2:n_col, with = FALSE])
-    rownames(readout) <- t(dfm[2:n_row, 1])
-    colnames(readout) <- dfm[1, 2:n_col]
+    readout <- dfm[2:n_row, 2:n_col, with = FALSE]
+    rownames(readout) <- as.character(t(dfm[2:n_row, 1]))
+    colnames(readout) <- as.character(dfm[1, 2:n_col])
```

Comment:
`data.table::setnames(readout, as.character(dfm[1, 2:n_col]))`

Reply:
Ok, here is the final version with the removed code for getting/setting rownames/colnames in data.table object.